.codeclimate.yml

```diff
@@ -40,4 +40,5 @@ exclude_patterns:
 - vendor/
 - app/api/search_consumer/
 -  "**/vendor/**/*"
+-  "**/vcr_cassettes/**/*"
```

Comment:
Question only: Why is this necessary? The vcr_cassettes are YAML files, which CC shouldn't be checking anyway.

Reply:
Code Climate runs 8 other engines in addition to Rubocop (see the list of `plugins`: https://github.com/GSA/search-gov/blob/master/.codeclimate.yml#L8). Some of those will check YAML files. For example, the `fixme` engine will flag stray `# TODO` or `# FIXME` comments in those files.